.travis.yml

```diff
@@ -12,6 +12,10 @@ matrix:
     env:
     - OS_NAME: macos
     osx_image: xcode10.2
+    cache:
+      directories:
+        - $HOME/Library/Caches/Homebrew
+        - /usr/local/Homebrew
```

Comment:
@hellais today [the build broke](https://travis-ci.org/ooni/probe-cli/jobs/582595891#L2446). Could this be related to attempts to create an homebrew cache? Can we confirm it works without cache?

Reply:
Yeah that would be a good thing to try out or as you were suggesting on #ooni-dev dropping macOS CI support entirely.

@bassosimone do you think you can pick this branch up and get it in a working state?

Reply:
Apart from dropping the macOS CI, anything else is required?

Reply:
I've filed #54 to address this specific issue in the PR.